app/config/assets_version.yml

```diff
@@ -1,2 +0,0 @@
-parameters:
-    assets_hash: d4b2bdcf09071f80c913
```

Comment:
This file is generated automatically by `make assets-prod` and should not be committed. 
So I removed it and is now only required for prod and test.